src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -143,71 +139,21 @@ internal void Rename(string oldPath, string newPath)
 
         #endregion
 
-        private class TrackerManager
```

Comment:
This class used to maintain the mapping between the file path and the single _IssueTracker_ for that file. No longer necessary.